test/workspaces/pipenv-app/Pipfile

```diff
@@ -6,10 +6,9 @@ name = "pypi"
 [packages]
 python-etcd = ">=0.4,<0.5"
 testtools = "*"
-"Jinja2" = { version = "<2.7" }
+"Jinja2" = { version = "*" }
 
 [dev-packages]
-virtualenv = "*"
```

Comment:
`virtualenv` was installed via `dev-requirements.txt`, so think it's not a good idea to reinstall it here which seems to mess up with things a bit